src/oemof/solph/flows/_flow.py

```diff
@@ -133,148 +145,104 @@ def __init__(self, **kwargs):
             "in version v0.6.\nRename the parameter to 'full_load_time_{0}', "
             "to avoid this warning and future problems. "
         )
-        if "summed_max" in kwargs:
+        if summed_max is not None:
             warn(msg.format("max"), FutureWarning)
-            kwargs["full_load_time_max"] = kwargs["summed_max"]
-        if "summed_min" in kwargs:
+            full_load_time_max = summed_max
+        if summed_min is not None:
             warn(msg.format("min"), FutureWarning)
-            kwargs["full_load_time_min"] = kwargs["summed_min"]
+            full_load_time_min = summed_min
         # --- END ---
 
         super().__init__()
 
-        scalars = [
-            "nominal_value",
-            "full_load_time_max",
-            "full_load_time_min",
-            "investment",
-            "nonconvex",
-            "integer",
-        ]
-        sequences = ["fix", "variable_costs", "min", "max"]
-        dictionaries = ["positive_gradient", "negative_gradient"]
-        defaults = {
-            "variable_costs": 0,
-            "positive_gradient": {"ub": None},
-            "negative_gradient": {"ub": None},
-        }
-        need_nominal_value = [
-            "fix",
-            "full_load_time_max",
-            "full_load_time_min",
-            "max",
-            "min",
-            # --- BEGIN: To be removed for versions >= v0.6 ---
-            "summed_max",
-            "summed_min",
-            # --- END ---
```

Comment:
Why is this missing below? It shall be removed from v0.6 onwards.

Reply:
I don't really get this comment, sorry.